spec/auth_helper.rb

```diff
@@ -0,0 +1,9 @@
+module AuthHelper
+  def http_login
+    user = 'user'
+    password = 'password'
+    {
+      HTTP_AUTHORIZATION: ActionController::HttpAuthentication::Basic.encode_credentials(user,password)
+    }
+  end
```

Comment:
What is the purpose of it? Could you please add actual logic for authentication?